dockerfiles/ocaml/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.10-alpine
+WORKDIR /go/src/github.com/sourcegraph/lsp-adapter
+COPY . .
+RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-adapter
+
+# 👀 Add steps here to build the language server itself 👀
+# CMD ["echo", "🚨 This statement should be removed once you have added the logic to start up the language server! 🚨 Exiting..."]
```

Comment:
should this be removed?

Reply:
https://github.com/sourcegraph/lsp-adapter/pull/28#discussion_r184816705